src/components/JokesCard.jsx

```diff
@@ -0,0 +1,20 @@
+import { 
+  Box
+} from "@chakra-ui/react";
+
+export default function JokesCard() {
+  return(
+    <Box 
+      as="strong" 
+      maxW='sm'    
+      bg='gray.100'
+      borderRadius={5}
+      p={8}
+      pl={16}
+    >
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aliquam in lorem sit amet leo accumsan lacinia. Etiam bibendum elit eget erat. 
```

Comment:
You can use `children` to pass down the content of the joke. Mock/stub data can be declared in the parent component.